bodyshopbidsdotcom/git-scripts
# w262783953 Vice Backend Production Environments

Closes https://www.wrike.com/open.htm?id=262783953

This enables the use of `--region <region_string>` when running `*_eb` commands and `*_terraform*` commands.

In addition, a [large amount of refactoring](https://github.com/bodyshopbidsdotcom/infra/pull/24) was needed in the infra repository. I updated the `git_terraform_commands` to reflect this.

### Testing

1. Change to your git-scripts directory.
1. Check out this branch.
1. Move to the vice-backend project directory.
1. Try the following commands:

```bash
$ git_terraform_commands --region eu-west-1 zurich

$ git_run_eb --region eu-west-1 zurich rake db:migrate

$ git_setenv_eb --region eu-west-1 zurich

$ git_push_eb --region eu-west-1 zurich
```

Ensure that default/existing environments also work:
```
# ...
$ git_run_eb qa1 rake db:migrate
# ...
```
